### PR TITLE
Translations - Add missing Simplified Chinese translation and change a few existing ones

### PR DESCRIPTION
**Pull request by mihuan-0**
**When merged this pull request will:**
- _Add missing Simplified Chinese translation and change a few existing ones
- _Each change in a separate line_

### IMPORTANT

- [x] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3mod.com/).
- [x] [Development Guidelines](https://ace3mod.com/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/arsenal/stringtable.xml [hunk 1]
```diff
@@ -1164,7 +1166,8 @@
             <Russian>Поддерживается доп. прицелом</Russian>
             <Portuguese>Secundária suportada</Portuguese>
             <French>Secondaire supportée</French>
-            <Chinese>次要武器支援</Chinese>
+            <Chinese>副武器支援</Chinese>
```
Comment (PabstMirror) [inline addons/arsenal/stringtable.xml:1169]:
```suggestion
            <Chinese>次要武器支援</Chinese>
```

File: addons/arsenal/stringtable.xml [hunk 2]
```diff
@@ -1164,7 +1166,8 @@
...
+            <Chinesesimp></Chinesesimp>
```
Comment (PabstMirror) [inline addons/arsenal/stringtable.xml:1170]:
```suggestion
            <Chinesesimp>副武器支援</Chinesesimp>
```

Comment (PabstMirror) [reply addons/arsenal/stringtable.xml:1170]: I think this is what you intended

Comment (mihuan-0) [reply addons/arsenal/stringtable.xml:1170]: yes, thanks!

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**